### DISCUSSION TITLE
Announcement: opentracing.io support for FastAPI

### DISCUSSION THREAD

**Discussion by acidjunk** [fastapi/fastapi#7568] [Questions]
Thanks for a great project. Porting some older Flask apps to FastAPI/Starlette as we speak.

As in some app's we used the tracing functionality of https://github.com/opentracing-contrib/python-flask and there didn't seem to exist an complete implementation for Starlette/FastAPI:

https://github.com/acidjunk/starlette-opentracing and https://pypi.org/project/Starlette-OpenTracing/

Feel free to close this ticket.

**Comment by tiangolo** [comment]
Hey @acidjunk , you could add the GitHub topic `fastapi` in your project repo, that would make it show up automatically in https://fastapi.tiangolo.com/external-links/#projects :smile: :coffee:

**Comment by acidjunk** [comment]
Thanks for the notification => Done. 😎
